.chloggen/correctness-traces.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: opencensusreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Check for ErrListenerClosed error before reporting multiplexer status.
```

Comment:
This message should be a bit more user-friendly. What does the user of the opencensus receiver care about, and understand?

Maybe something like "Don't report an error for an expected condition during shutdown"?